staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```diff
@@ -26,6 +27,7 @@ import (
 	"strings"
 
 	"github.com/imdario/mergo"
+	"go.etcd.io/etcd/client/pkg/v3/fileutil"
```

Comment:
not too happy about this ... our client-go depending on "go.etcd.io/etcd/client/pkg/v3/fileutil"

cc @liggitt

Reply:
It's already in other areas of the codebase: https://github.com/kubernetes/kubernetes/blob/master/go.mod#L80

Reply:
that's not really relevant... adding dependencies to k8s.io/{api,apimachinery,client-go} pushes them onto far more consumers